test/test_cart_shift.jl

```diff
@@ -0,0 +1,22 @@
+using Test
+using MPI
+
+MPI.Init()
+comm = MPI.COMM_WORLD
+nnodes = MPI.Comm_size(comm)
+ndims = 3
+reorder = 1
+periods = [0,1,0]
+dims = [0,0,0]
+MPI.Dims_create!(nnodes, dims)
+comm_cart = MPI.Cart_create(comm, dims, periods, reorder)
+coords = MPI.Cart_coords!(comm_cart, ndims)
+disp = 1
+
+for i in 0:2
+    neighbors = MPI.Cart_shift(comm_cart, i, disp)
+    # TODO: replace -1 with MPI.PROC_NULL
+    @test all( ((neighbors .>= 0) .& (neighbors .< nnodes)) .| (neighbors .== -1) )
```

Comment:
Can you tell me what I need to do to get `MPI_PROC_NULL` available as `MPI.PROC_NULL`? Adding `MPI_PROC_NULL` to `deps/gen_constants.f90` was somehow not enough...

Reply:
Most constants show up as "MPI.MPI_CONST_NAME", (which is a bit redundent).  `MPI.COMM_WORLD` and the `MPI_Op` constants are the exceptions.  Can you check `deps/src/compile-time.jl` for `MPI_PROC_NULL`?  If it is there, it should be available for use.